db_scripts/init_db.sh

```diff
@@ -7,7 +7,7 @@ if [ $# -ge 1 ] ; then
 fi
 
 NS=compliance-toolkit
-CHART_VERSION=12.0.2
+CHART_VERSION=12.0.2-B4
```

Comment:
this should be 12.0.1-B4

Reply:
it  should be 12.0.1-B4 because we released postgres-init with 1.2.0.1-B4